tensorflow/core/kernels/quantize_op_test.cc

```diff
@@ -14,37 +14,58 @@ limitations under the License.
 ==============================================================================*/
 
 #include <random>
+#include <stdexcept>
```

Comment:
What is this needed for?  We build with exceptions disabled.

Reply:
Removed. It was an artifact of clang-format on my end.